packages/dx-react-chart/src/types/templates.axis.types.ts

```diff
@@ -0,0 +1,37 @@
+export interface LabelProps {
+  // The label text
+  text: string | number;
```

Comment:
Why not just `string`?

Reply:
I get it from our [documentation](https://devexpress.github.io/devextreme-reactive/react/chart/docs/reference/value-axis/). I think it is because `tick` can be formatted or returned as it is (number): https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-chart-core/src/plugins/axis/computeds.ts#L18